frontend/src/scenes/onboarding/home/sections/DiscoverInsight.tsx

```diff
@@ -156,7 +156,7 @@ function RecentInsightList(): JSX.Element {
                             <Divider />
                             <Paragraph>Jump back into recent work or an analysis from one of your teammates.</Paragraph>
 
-                            <InsightHistoryPanel displayLocation="project home" />
+                            <InsightHistoryPanel id="discover" displayLocation="project home" />
```

Comment:
This isn't quite the right solution in its current form, because then the logic thinks we're trying to load a dashboard ID "new", and no such thing exists.
Here's what happens when I visit Home now:
<img width="408" alt="Err" src="https://user-images.githubusercontent.com/4550621/128838120-e56e26f1-2122-4db5-b45c-9ec0645656b4.png">

Reply:
nice catch, maybe it makes sense to use default '' prop since it looks like prop.id will always be synonymous as the dashboardId. I'll make this fix in the next iter